NAMESPACE

```diff
@@ -5,30 +5,40 @@ S3method(docdb_create,src_elastic)
 S3method(docdb_create,src_etcd)
 S3method(docdb_create,src_mongo)
 S3method(docdb_create,src_redis)
+S3method(docdb_create,src_sqlite)
 S3method(docdb_delete,src_couchdb)
 S3method(docdb_delete,src_elastic)
 S3method(docdb_delete,src_etcd)
 S3method(docdb_delete,src_mongo)
 S3method(docdb_delete,src_redis)
+S3method(docdb_delete,src_sqlite)
 S3method(docdb_exists,src_couchdb)
 S3method(docdb_exists,src_elastic)
 S3method(docdb_exists,src_etcd)
+S3method(docdb_exists,src_mongo)
```

Comment:
remove those mongo changes in the NAMESPACE

Reply:
Corrected, thank you.